systems/overview/12-Glossary.md

```diff
@@ -55,3 +55,15 @@ MediaWiki Core refers to all files that are part of the main MediaWiki package.
 ## [Special Page](https://www.mediawiki.org/wiki/Manual:Special_pages)
 
 Special pages are dynamically created MediaWiki pages, which perform a specific function, such as providing a list of pages, showing statistics or creating a form for user submitted feedback. They are located in their own namespace (_Special:_) and are not editable directly like other pages.  Developers can create new special pages. They will generally show up in the list of all special pages at [Special:SpecialPages](https://www.wikidata.org/wiki/Special:SpecialPages).
+
+## Gadgets
+
+Gadgets are [tools](#tools) which can be enabled from you Wikidata's account preferences.
```

Comment:
Hm, I tried to find a link to an "official" description of what Gadgets are, but couldn't find any. Maybe we can add a sentence to mention that they're enabled via https://www.mediawiki.org/wiki/Extension:Gadgets

Reply:
Yep, me neither, so I came up with this sentence. I'm not even 100% sure that gadgets and tools are synonyms in Wikidata's context.
Will add the link you mentoned.

Reply:
I like the distinction we've seen in the Mattermost chat and made by @manicki (thanks!), i.e. Gadgets = "tools" that run on the wiki, tools = external applications or scripts.

We can also steal some of the description from https://www.mediawiki.org/wiki/Extension:Gadgets :
```suggestion
[Gadgets](https://www.mediawiki.org/wiki/Extension:Gadgets) are made up of JavaScript and/or CSS Snippets located on pages in the MediaWiki namespace. These snippets add functionality to the wiki itself and can be enabled and disabled via preferences.
```